### PR TITLE
Add back `buildType: nixpacks` per Flightcontrol request

### LINKED ISSUE #7597: [Bug?]: SyntaxError: Unexpected token '??=' after upgrade to Redwood 4 + changed engine to node > 16
### What's not working?

👋 Currently trying to deploy to Flightcontrol per usual after upgrading to Redwood 4. Seems like it's not liking the null coalescing assignment [here](https://github.com/redwoodjs/redwood/blame/8529ef8ff48f4fb2b4c5c4b81fd5a0f0359d7fe4/packages/cli/src/index.js#L60) that has only gained support in node > 15. I added `"node": ">=16.x <=18.x",` under engines to my `package.json` but it still doesn't seem to be functional.

```
[Container] 2023/02/08 23:11:47 Running command (yarn rw deploy flightcontrol web)
6:11:48 PM: /codebuild/output/src513646770/src/node_modules/@redwoodjs/cli/dist/index.js:63
6:11:48 PM: cwd ??= process.env.RWJS_CWD;
6:11:48 PM:     ^^^
6:11:48 PM: 
6:11:48 PM: SyntaxError: Unexpected token '??='
6:11:48 PM:     at wrapSafe (internal/modules/cjs/loader.js:1001:16)
6:11:48 PM:     at Module._compile (internal/modules/cjs/loader.js:1049:27)
6:11:48 PM:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
6:11:48 PM:     at Module.load (internal/modules/cjs/loader.js:950:32)
6:11:48 PM:     at Function.Module._load (internal/modules/cjs/loader.js:790:12)
6:11:48 PM:     at Module.require (internal/modules/cjs/loader.js:974:19)
6:11:48 PM:     at require (internal/modules/cjs/helpers.js:101:18)
6:11:48 PM:     at Object.<anonymous> (/codebuild/output/src513646770/src/node_modules/@redwoodjs/core/dist/bins/redwood.js:30:1)
6:11:48 PM:     at Module._compile (internal/modules/cjs/loader.js:1085:14)
6:11:48 PM:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
```

Builds fine locally as really all that the web site is doing is `yarn rw build web` based upon [this code](https://github.com/redwoodjs/redwood/blob/main/packages/cli/src/commands/deploy/flightcontrol.js#L79).

Wondering if any ideas?

I'll try a completely new build on Flightcontrol soon. I've already reached out on their discord. Thanks!

### How do we reproduce the bug?

_No response_

### What's your environment? (If it applies)

_No response_

### Are you interested in working on this?

- [ ] I'm interested in working on this

### PR DESCRIPTION
**Pull request by schybo**
Following up from the [issue here ](https://github.com/redwoodjs/redwood/issues/7597#issuecomment-1424054130) where the default `flightcontrol.json` no longer works after upgrading to RedwoodJS 4. Seems like the `buildType: 'nixpacks'` option was removed in [this PR](https://github.com/redwoodjs/redwood/pull/6941) (unsure why it was - seems the [issue](https://github.com/redwoodjs/redwood/issues/6927) it was solving only needed the `postBuildCommand` be removed.

Regardless, after discussing with the Flightcontrol team, we needed to get that added back for builds to work such that the node version would be respected and not fail on the null coalescing operator that was only introduced in node 15.

tl;dr - Closes #7597

### CODE REVIEW

File: packages/cli/src/commands/setup/deploy/templates/flightcontrol.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ export const flightcontrolConfig = {
           id: 'redwood-web',
           name: 'Redwood Web',
           type: 'static',
+          buildType: 'nixpacks',
           singlePageApp: true,
           installCommand:
             'yarn set version stable && NODE_ENV=development yarn install',
```
Comment (flybayer) [inline packages/cli/src/commands/setup/deploy/templates/flightcontrol.js:36]:
```suggestion
```

**Review by flybayer — CHANGES_REQUESTED** [review]
awesome! We had to remove it before because it was unsupported at that time.

**Review by flybayer — APPROVED** [review]

**Comment by jtoar** [comment]
@schybo this fix was just released in v4.1.2 https://github.com/redwoodjs/redwood/releases/tag/v4.1.2

**Comment by schybo** [comment]
Thanks @jtoar
